wire-ios-request-strategy/Sources/Object Syncs/Helpers/ZMChangeTrackerBootstrap.m

```diff
@@ -84,6 +83,20 @@ - (void)fetchObjectsForChangeTrackers
     }
 }
 
+- (void)fetchObjectsForChangeTrackers
+{
+    [self fetchObjectsForChangeTrackers:self.changeTrackers];
+}
+
+- (void)addChangeTrackers:(NSArray *)changeTrackers
+{
+    [self fetchObjectsForChangeTrackers:changeTrackers];
```

Comment:
fetch objects for newly added changeTrackers (ClientMessageRequestStrategy, Asset...)